app/webpacker/styles/course_lesson.scss

```diff
@@ -0,0 +1,7 @@
+.duration h2 {
+  margin-bottom: 50px;
+}
+
+.lesson-heading {
+  margin-bottom: 20px;
+}
```

Comment:
Hehe, let's use govuk design system for this. Even if they are not as precise as we want.

`govuk-!-margin-bottom-8` for `duration`, `govuk-!-margin-bottom-4` for `lesson-heading` :) 

https://design-system.service.gov.uk/styles/spacing/